File: latihanA/Ram.cpp
#include <bits/stdc++.h>

using namespace std;

class Ram {
    private:
        int capacity;
        int price;
    
    public:
        Ram() {
            this->capacity = 0;
            this->price = 0;
        }

        Ram(int capacity, int price) {
            this->capacity = capacity;
            this->price = price;
        }

        void setCapacity(int capacity) {
            this->capacity = capacity;
        }

        int getCapacity() {
            return this->capacity;
        }

        void setPrice(int price) {
            this->price = price;
        }

        int getPrice() {
            return this->price;
        }

        void printRam() {
            cout << "Capacity\t: " << this->capacity << " GB" << endl;
            cout << "Price\t\t: " << this->price << " IDR" << endl;
        }

        ~Ram() { }
};

File: latihanB/python/Ram.py
class Ram:
    def __init__(self, capacity=0, price=0):
        self.__capacity = capacity
        self.__price = price

    def setCapacity(self, capacity=0):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity
    
    def setPrice(self, price=0):
        self.__price = price

    def getPrice(self):
        return self.__price

    def printRam(self):
        print("Capacity\t: " + str(self.__capacity) + " GB")
        print("Price\t\t: " + str(self.__price) + " IDR")